src/server/electron/electron.ts

```diff
@@ -58,7 +58,7 @@ export class ElectronApplication extends SdkObject {
   private _nodeConnection: CRConnection;
   private _nodeSession: CRSession;
   private _nodeExecutionContext: js.ExecutionContext | undefined;
-  _nodeElectronHandle: js.JSHandle<any> | undefined;
+  _nodeElectronHandle: Promise<js.JSHandle<any> | undefined>;
```

Comment:
`Promise<js.JSHandle<any>> | undefined;`

Reply:
I'm not very sure whether '_nodeElectronHandle' will be reached before '_init' has finished or not, I just find '_onPage' and 'close' in server part. 

It looks that '_init' will always finish before 'onPage' when I have tested this PR in my application. 

If we wanna ensure '_nodeElectronHandle' to be a Promise, maybe we could extract its initialization to a new method like 'initNodeHandle', and await launch process at top of it?

Reply:
Electron support uses two independent connections, one going to Node and another going to the browser. These connections race naturally and if you throttle the Node one, you can see how browser talks to us before node handle is initialized. You can add `await new Promise(f => setTimeout(f, 1000));` before `await this._nodeSession.send('Runtime.enable', {}).catch(e => {});` in the init to trigger it. You last push should resolve it! Let's see if it fixes my local test and if it does, we merge it. I'll follow up with the test and a small refactoring.

Reply:
@pavelfeldman Hope it works! I found some eslint issues in my commit, just fix and amend it to the only commit.